alarm/src/index.ts

```diff
@@ -22,9 +22,9 @@ const server: HttpServer = http.createServer(app)
 
 app.use((req: Request, res: Response, next: NextFunction) => {
   const authHeader = req.headers.authorization
-  const token = authHeader && authHeader.split(' ')[1]
+  const token = authHeader && authHeader.split(' ')[1] || ''
 
-  if (token === process.env.DEV_API_KEY) return next()
+  if (jwtManager.admittedTokens().includes(token)) return next()
```

Comment:
after the if to check if the token is one of the admitted you have to put 
  if (token === undefined || token === '') return res.status(403).send({ error: 'No authentication token' })
  (like in other middlewares)
  to check the token. Adding this line the test that now fail in CI should pass.